package.json

```diff
@@ -72,7 +72,7 @@
   "dependencies": {
     "@americanexpress/env-config-utils": "^2.0.2",
     "@americanexpress/lumberjack": "^1.1.3",
-    "@americanexpress/one-app-bundler": "^6.4.0",
+    "@americanexpress/one-app-bundler": "^6.5.0",
```

Comment:
looks to be added as a dependency provided by the one-app runtime
https://github.com/americanexpress/one-app-cli/pull/59/files#diff-6bf1566aa3c949a6604b1e1f1750e2a07bde2a1ba5f10a7c39acfe05332e8816